services/web/server/src/simcore_service_webserver/users/_preferences_api.py

```diff
@@ -57,6 +57,20 @@ async def _get_frontend_user_preferences(
     ]
 
 
+async def get_user_preference(
+    app: web.Application,
+    user_id: UserID,
+    product_name: ProductName,
+    preference_class: AnyUserPreference,
+) -> AnyUserPreference | None:
+    return await _preferences_db.get_user_preference(
+        app,
+        user_id=user_id,
+        product_name=product_name,
+        preference_class=preference_class,
+    )
```

Comment:
I've already said this. I don't see the point of adding a redirection to this call. I would just import it in the `preferences_api.py` module directly from the `_prefernces_db.py`

Reply:
Yes, we discussed this in person. I like to always "decouple" these, so I do not expose database objects outside. The advantage of this approach is we something changes in the database, for example, if we add a new column, I do not need to mess up a contract with "outside package". Here is the abstract layer where I can have it under control.